examples/src/bin/wifi_embassy_bench.rs

```diff
@@ -100,19 +98,13 @@ async fn main(spawner: Spawner) -> ! {
     #[cfg(feature = "esp32")]
     {
         let timg1 = TimerGroup::new(peripherals.TIMG1, &clocks);
-        let timer0: ErasedTimer = timg1.timer0.into();
-        let timers = [OneShotTimer::new(timer0)];
-        let timers = mk_static!([OneShotTimer<ErasedTimer>; 1], timers);
-        esp_hal_embassy::init(&clocks, timers);
+        esp_hal_embassy::init(&clocks, timg1.timer0);
```

Comment:
I wonder, could we use timg1.timer1 here? If yes, I'll update the examples because it doesn't feel right to recommend code that "wastes" timers.

Reply:
Did you mean `timg0.timer1`? i.e. Use the same timer group for both WiFi and embassy.

@bjoernQ's [intention](https://github.com/esp-rs/esp-hal/pull/1753#issuecomment-2210279380) with the examples was to showcase different initialization patterns. So making some of the examples use only TIMG0 and some use both TIMG0 and TIMG1, probably makes sense to follow that.

Though given that TIMG [suffers](https://github.com/esp-rs/esp-hal/blob/v0.19.0/esp-hal/src/timer/timg.rs#L724-L740) from #1874, "wasting" a timer might be better recommendation for now.

Reply:
Yes I meant `timg0.timer1`, thank you.